### PR TITLE
Add CLI option to not realign BAM samples

### LINKED ISSUE #138: Add a --no-realign option: if BAMs are given, don't convert them to FQs first

### PR DESCRIPTION
**Pull request by ihodes**
relies on https://github.com/hammerlab/biokepi/pull/409

fixes https://github.com/hammerlab/epidisco/issues/138

### CODE REVIEW

File: src/lib/command_line.ml [hunk 1]
```diff
@@ -281,9 +264,53 @@ let args pipeline =
...
+let sample_args transform_term =
+  let open Cmdliner in
+  let open Cmdliner.Term in
+  let sample_files_arg ?(req = true) option_name f =
+    let doc =
+      sprintf
+        "PATH/URI(s) to data files (BAM, FASTQ, FASTQ.gz) or serialized \
+         sample description in JSON format for the %S sample. \
+         Use comma (,) as a delimiter to provide multiple data files \
+         and semi-colon (@) when describing paired-end FASTQ files."
```
Comment (smondet) [inline src/lib/command_line.ml]: I know it was already there, but `@` is not a semi-colon :)

File: src/lib/command_line.ml [hunk 2]
```diff
@@ -281,9 +264,53 @@ let args pipeline =
...
+  transform_term
   $ sample_files_arg "normal" (fun s -> `Normal_sample s)
   $ sample_files_arg "tumor" (fun s -> `Tumor_sample s)
   $ sample_files_arg ~req:false "rna" (fun s -> `Rna_sample s)
+
+let args pipeline =
+  let open Cmdliner in
+  let open Cmdliner.Term in
+  let ( <.> ) a b =
+    b a
```
Comment (smondet) [inline src/lib/command_line.ml]: Why not `|>`?

**Review by smondet — COMMENTED** [review]
LAGTM ("Almost Good" :) )

File: src/lib/command_line.ml (hunk 1 above)
Comment (ihodes) [reply src/lib/command_line.ml]: Fixed!

File: src/lib/command_line.ml (hunk 2 above)
Comment (ihodes) [reply src/lib/command_line.ml]: 🦀

**Comment by ihodes** [comment]
Back to you!

**Pull request merged**